packages/components/src/modal/Modal.tsx

```diff
@@ -73,7 +73,8 @@ function Modal({
         onOpened();
       }
     },
-    [onOpened, isOpen]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [isOpen]
```

Comment:
Returned to its original version.

Currently staying with Modal because Dialog uses a different background color which makes monaco's background look jarring to me